themes/root/templates/Helpers/icons/font.phtml

```diff
@@ -7,4 +7,4 @@
         $this->attrs = preg_replace('/ class="[^"]+"/', '', $this->attrs);
     }
 ?>
-<span class="icon--font <?=$this->escapeHtmlAttr($this->icon) ?>"<?=$this->attrs ?> role="img" aria-hidden="true"></span>
+<span class="icon--font <?=$this->icon ?>"<?=$this->attrs ?> role="img" aria-hidden="true"></span>
```

Comment:
Was the removal of escaping here intentional? Is proper escaping guaranteed elsewhere?

Reply:
It was problematically double-escaping bc it also escapes this in the Helper.